lizmap/modules/lizmap/lib/Project/Project.php

```diff
@@ -218,6 +223,17 @@ public function __construct($key, \LizmapRepository $rep, App\appContextInterfac
                 if (array_key_exists($prop, $data)) {
                     $this->{$prop} = $data[$prop];
                 }
+                foreach ($this->layers as $index => $layer) {
+                    if ($layer['embedded'] == '1' && $layer['qgsmtime'] < filemtime($layer['file'])) {
+                        $qgsProj = new QgisProject($layer['file']);
```

Comment:
why not. The issue here, is that the cache is not updated. So if one of qgis project is modified, it will reread it each time Project is instantiated.